enterprise/apps.py

```diff
@@ -14,3 +14,5 @@ class EnterpriseConfig(AppConfig):
     """
 
     name = 'enterprise'
+    valid_extensions = ['.png', ]
+    image_size = 250 * 1024
```

Comment:
@mattdrayer We need to finalize the size of enterprise customer logo image size. I just used it for the time being but it is not the ideal size. anyone else to tag ?